### PR TITLE
update pattern some more

### PR DESCRIPTION
**Pull request by stevemar**
* add a real gitignore
* remove jshint, it's not used
* remove configs for eshint, the defaults work fine
* remove codecov libraries and travis call, there are no tests, so no reason to have this for now
* remove IDE specific files
* add a missing license header
* move the data to a top level folder
* remove `AWAPlinuxonecc/demoCloudScripts/myUserDataSample1.sh` cause it's mysterious and it's not used throughout the pattern

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,92 @@
...
+.vuepress/dist
+
+# Serverless directories
+.serverless/
+
+# FuseBox cache
+.fusebox/
+
+# DynamoDB Local files
+.dynamodb/
```
Comment (rhagarty) [inline .gitignore]: should probably add `.DS_Store` to list

File: README.md [hunk 2]
```diff
@@ -34,21 +34,21 @@ When you will complete this code pattern, you will understand how to:
 
 ## Steps
 
-1. [Create an IBM Watson Assistant Service & Configure it]()
-1. [Build and deploy a Docker image to IBM Cloud private]()
-1. [Instantiate the provisiong chatbot from the IBM Cloud private catalog]()
-1. [Run the application: Talk and get it done with IBM LinuxONE Systems]()
-1. [Extend this solution to access your on-premises servers]()
+1. [Create and configure an Watson Assistant]()
```
Comment (rhagarty) [inline README.md]: Replace `an` with `a` or just remove

File: README.md [hunk 3]
```diff
@@ -34,21 +34,21 @@ When you will complete this code pattern, you will understand how to:
...
+1. [Build and deploy a Docker image to IBM Cloud Private]()
+1. [Instantiate the chatbot from the IBM Cloud Private catalog]()
+1. [Run the application]()
+1. [Extend this solution]()
 
-### 1. Create an IBM Watson Assistant Service
+### 1. Create and configure an Watson Assistant
```
Comment (rhagarty) [inline README.md]: this needs to match step name above

File: README.md [hunk 4]
```diff
@@ -321,7 +321,7 @@ The objective is to discover the IBM Cloud private catalog in order to instantia
 :thumbsup: Congratulations! Your Cognitive Hybrid Cloud application has been instantiated from IBM Cloud Private as container. Your Cognitive Hybrid Cloud application succeeded to execute a command in your LinuxONE Server.
 
 ---
-### Step 5 - Extend this solution to access your on-premises servers
+### 5. Extend the solution
```
Comment (rhagarty) [inline README.md]: name needs to match step name above

**Comment by stevemar** [comment]
@rhagarty i'll update the README tonight.

**Pull request merged**